soliket/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .constants import *
```

Comment:
I'd prefer not to have the constants live at the top package level.  Not a strong preference, but I think it's fine to explicitly import the constants when needed.